frontend/src/components/NavBar.js

```diff
@@ -4,16 +4,18 @@ import '../css/NavBar.css'
 
 const NavBar = props => (
     <div className='Nav-Bar'>
-        <div className='nav-logo-container'>
-            <div className='icon-border'>
-                <img className='nav-icon' src='https://image.flaticon.com/icons/svg/1530/1530648.svg' alt = 'whisk icon'/>
-            </div>  
-            <h1 className='nav-logo'>k</h1>
-        </div>
-        <div className='nav-link-container'>
-            <Link className='nav-link' to='/create'>Add Recipe</Link>
-            <Link className='nav-link' to="/settings">Settings</Link>
-            <Link className='nav-link' to="/billing">Billing</Link>
+       <div className='nav-link-overlay'>
+            <div className = 'nav-link-container'>
+                <Link className='nav-link' to='/create'>Create</Link>
+                <Link className='nav-link' to="/recipes">Recipes</Link>
+                <Link className='nav-link' to="/grocerylist">Grocery List</Link>
```

Comment:
This is entirely up to you, but if there are going to be multiple words in the url, separate with a `-` 
```suggestion
                <Link className='nav-link' to="/grocery-list">Grocery List</Link>
```
That is up to you and your group though